client/src/consts/theme.tsx

```diff
@@ -1,11 +1,30 @@
 const colors = {
   gray100: "#393838",
+  gray210: "#343434",
+  gray220: "#525252",
+  gray230: "#6e6e6e",
+
+  black100: "#03030363",
+  black110: "#131313",
 } as const;
 
 export const theme = {
   button: {
     backgroundColor: colors.gray100,
   },
+  dropdownContainer: {
+    backgroundColor: colors.gray210,
+  },
+  dropodwnStyle: {
+    border: colors.gray220,
+    boxshadow: colors.black100,
+    backgroundColor: colors.gray210,
+  },
+  dropdownIcon: {
+    border: colors.gray220,
+    dropShadow: colors.black110,
+    hoverbackgroundColor: colors.gray230,
+  },
```

Comment:
Instead of adding all of the properties to the toplevel, it's a better idea to make it nested like this:


```suggestion
  dropdown: {
    container: {
      backgroundColor: colors.gray210,
    },
    style: {
      border: colors.gray220,
      boxshadow: colors.black100,
      backgroundColor: colors.gray210,
    },
    icon: {
      border: colors.gray220,
      dropShadow: colors.black110,
      hoverbackgroundColor: colors.gray230,
    },
  },
```

The whole value behind nesting is the conciseness in naming and organising stuff. Instead of doing `containerFoo` and `containerBar`, far better is to use the `container.foo` and `container.bar`.